spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/TaskServiceUtilsTests.java

```diff
@@ -53,10 +53,15 @@
 	public void testCreateComposedTaskDefinition() {
 		TaskConfigurationProperties props = new TaskConfigurationProperties();
 		props.setComposedTaskRunnerName("FOO");
-		TaskServiceUtils.createComposedTaskDefinition(BASE_GRAPH, props);
 		assertThat(TaskServiceUtils.createComposedTaskDefinition(BASE_GRAPH, props)).isEqualTo("FOO --graph=\"AAA && BBB\"");
 	}
 
+	@Test
+	public void testCreateComposedTaskDefinitionWithUserCTRName() {
```

Comment:
how does this test make assertions on a custom ctr name?

Reply:
Actually I can delete this.   Replacing a CTR name only occurs during the task launch.   Good catch.